src/wallet/mod.rs

```diff
@@ -2077,6 +2077,10 @@ pub(crate) mod test {
         "tr(cNJmN3fH9DDbDt131fQNkVakkpzawJBSeybCUNmP1BovpmGQ45xG,{pk(tprv8ZgxMBicQKsPdDArR4xSAECuVxeX1jwwSXR4ApKbkYgZiziDc4LdBy2WvJeGDfUSE4UT4hHhbgEwbdq8ajjUHiKDegkwrNU6V55CxcxonVN/*),pk(8aee2b8120a5f157f1223f72b5e62b825831a27a9fdf427db7cc697494d4a642)})"
     }
 
+    pub(crate) fn get_test_tr_dup_keys() -> &'static str {
```

Comment:
Could this string just be added to `test_taproot_load_descriptor_duplicated_keys` or do you anticipate using this descriptor in other tests?

Reply:
I think it could be used in other tests, for example if we want to see how the policy module deals with duplicated keys